homeassistant/components/mysensors/__init__.py

```diff
@@ -110,8 +108,6 @@ def validator(config):
                     vol.Required(CONF_GATEWAYS): vol.All(
                         cv.ensure_list, has_all_unique_files, [GATEWAY_SCHEMA]
                     ),
-                    vol.Optional(CONF_OPTIMISTIC, default=False): cv.boolean,
```

Comment:
Wts keep these validators but also add `cv.deprecated` to mark the items as deprecated.